CHANGELOG

```diff
@@ -1,3 +1,7 @@
+BETA
+-----
+* Add WebUSB support for direct flashing and a serial connection to the REPL.
+
```

Comment:
Now that we have a beta branch perhaps we should move this edit to the change from this feature branch into the beta branch.
This was there will be less conflicts when merging to master with other feature branches.